ariadne/tracknet_v2_2/dataset.py

```diff
@@ -0,0 +1,114 @@
+from __future__ import print_function, division
+import os
+import torch
+import pandas as pd
+import numpy as np
+from torch.utils.data import Dataset, DataLoader
+from copy import deepcopy
+import gin
+from ariadne.tracknet_v2.model import TrackNETv2
+from ariadne.tracknet_v2.dataset import TrackNetV2Dataset
+from ariadne.tracknet_v2.metrics import point_in_ellipse
+from ariadne.utils import weights_update, find_nearest_hit, load_data
+import warnings
+warnings.filterwarnings("ignore")
+
+
+@gin.configurable    
+class TrackNetV22Dataset(TrackNetV2Dataset):
```

Comment:
Is it a copy of a class from tracknet_v2_1?

Reply:
no, it is slightly different, because we create input features from coordinates, not gru state of tracknetv2. Commented it in docstrings.